### PR TITLE
update workshop documentation for onboarding, github setup and visualization dashboards

### PR DESCRIPTION
**Pull request by aakankshaduggal**
## Related Issues and Dependencies

Related to #540 
Related to #546

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

… Explain your changes.

### CODE REVIEW

File: docs/workshop/visualization_dashboard.md [hunk 1]
```diff
@@ -1,6 +1,69 @@
-# Visualization Dashboard - Superset
+# Visualization Dashboard
 
-This section will consist Superset visualization dashboard :
+After finding key performance indicator metrics, developing and deploying machine learning models, we will create interactive visualization dashboards by using Apache Superset.
 
-* Import a dataset using the table added on Trino.
-* Create charts and dashboard
+[Superset](https://superset-aiops-tools-workshop.apps.smaug.na.operate-first.cloud/) is an open source visualization tool that allows users to create interactive dashboards. Superset is very simple to use and may require some basic knowledge about SQL.
+
```
Comment (oindrillac) [inline docs/workshop/visualization_dashboard.md:6]:
```suggestion
For this section, you will need access to [Apache Superset](https://superset-aiops-tools-workshop.apps.smaug.na.operate-first.cloud/) to create charts and dashboards. If you dont have access already, refer to login instructions in the [onboarding chapter](./onboarding.md).
```

File: docs/workshop/visualization_dashboard.md [hunk 2]
```diff
@@ -1,6 +1,69 @@
...
+Here's how you can access the Superset, create your own charts and dashboards.
+
+1. First start by going to [Superset](https://superset-aiops-tools-workshop.apps.smaug.na.operate-first.cloud/) page and log in by clicking on `operate-first` using your github account.
+
+![alt_text](../assets/images/superset_landing.png "image_tooltip")
+
+2. After logging into Superset you will have to import the datasets you would want to visualize. Start by clicking on `Data` -> `Datasets` -> click on `+ Datasets` to start importing the data to superset from trino Database.
+
```
Comment (oindrillac) [inline docs/workshop/visualization_dashboard.md:14]:
```suggestion

Import the dataset that you created in the [previous step](./sql_query_engine.md), it should be set to your username or have the same name that you set your environment variable `CEPH_BUCKET_PREFIX`.
```

File: docs/workshop/visualization_dashboard.md [hunk 3]
```diff
@@ -1,6 +1,69 @@
...
+* This is how the dashboard would first look like, edit the `[ untitled dashboard]` to name it.
+
+![alt_text](../assets/images/superset-dash2.png "image_tooltip")
+
+* Now, in order to import the chart we created earlier, go to `charts` like shown below and select the chart you created, drag it and place it in the dashboard.
+
+![alt_text](../assets/images/superset-dash3.png "image_tooltip")
+
+Here's how you dashboard would look like -
```
Comment (oindrillac) [inline docs/workshop/visualization_dashboard.md:50]:
```suggestion
Here's how your dashboard would look like -
```

File: docs/workshop/visualization_dashboard.md [hunk 4]
```diff
@@ -1,6 +1,69 @@
...
+![alt_text](../assets/images/superset-dash5.png "image_tooltip")
+
+![alt_text](../assets/images/superset-dash6.png "image_tooltip")
+
+* And finally to export the dashboard you created, go to export a dashboard first go to the `Dashboards` tab of the top bar.  Next, hover under the `Actions` column next to the dashboard you would like to export. Finally, click on the `arrow icon` to export the dashboard and a JSON file download should start. Make sure that you have pop-ups enabled for this page or the download might not run.
+
+![alt_text](../assets/images/superset-dash7.png "image_tooltip")
+
+
+And that's it you now have a vizualization chart and a dashboard ready.
```
Comment (oindrillac) [inline docs/workshop/visualization_dashboard.md:64]:
```suggestion
And that's it, now you have learnt how to create visualizations in Apache Superset using tables created from data collected from a Github repository.
```

File: docs/workshop/visualization_dashboard.md [hunk 5]
```diff
@@ -1,6 +1,69 @@
-# Visualization Dashboard - Superset
+# Visualization Dashboard
 
-This section will consist Superset visualization dashboard :
+After finding key performance indicator metrics, developing and deploying machine learning models, we will create interactive visualization dashboards by using Apache Superset.
```
Comment (oindrillac) [inline docs/workshop/visualization_dashboard.md:3]:
```suggestion
So far we have seen how to collect Pull Requests data from a Git repository, processing the data, develop and deploy AIOps models. Now we will learn how to create interactive visualization dashboards using Apache Superset.
```

File: docs/workshop/git_setup.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
 # Set up Github environment
 
-## Create Github account, Fork Repository, Generate Github Access token
+In order to get started with the workshop, it's vital to have a github account, so you can also interact with the codebase and make changes as and when required. Here are a few steps that you need to follow along to be all set to get started with this workshop.
 
-* **Create a Github account**: Go to[ github.com](https://github.com/) and click on sign up to create a new account.
+* **Create a Github account**: Go to[ github.com](https://github.com/) and click on `Sign Up` to create a new account.
```
Comment (oindrillac) [inline docs/workshop/git_setup.md:6]: All images in this doc look quite unclear/low quality. Can these be updated?

**Review by oindrillac — CHANGES_REQUESTED** [review]

**Comment by oindrillac** [comment]
Looks good! @aakankshaduggal  Suggested some minor changes :)

File: docs/workshop/git_setup.md (hunk 6 above)
Comment (aakankshaduggal) [reply docs/workshop/git_setup.md:6]: Let me add them again.

**Review by suppathak — COMMENTED** [review]
Great Documentation. :) :rocket: 
Thanks @aakankshaduggal 
/lgtm

**Review by oindrillac — APPROVED** [review]
/lgtm
/approve

**Pull request merged**